src/app/models/indicator-query-opts.model.ts

```diff
@@ -7,8 +7,10 @@ export interface IndicatorQueryOpts {
     indicator: Indicator;
     city: City;
     scenario: Scenario;
-    climateModels?: ClimateModel[];
-    years?: string[];
-    time_aggregation?: string;
-    unit?: string;
+    params: {
```

Comment:
There are the query parameters passed to make the request and not intended to map to the response. Intention here is to distinguish them from the *path* parameters, which are the other things on this object.